nominations/models.py

```diff
@@ -355,6 +470,76 @@ def save(self, *args, **kwargs):
         if self.vol_notes and not self.staff_notes:
             self.staff_notes = self.vol_notes
 
+    def create_related_objects(self):
+        """Create related nomination and questionnaire for application."""
+        if not self.nomination:
+            self.nomination = Nomination.objects.create()
+
+        if not self.questionnaire:
+            self.questionnaire = Questionnaire.objects.create()
+
+    def generate_application_status(self):
+        """Returns a generated status based on completion of various items."""
+
+        if self.nomination.status == 'incomplete':
```

Comment:
Will add comments on the nomination model explaining what it is. In future we can move under application but for now the reason we have two statuses is because they're separate models.